### DISCUSSION TITLE
What "Expose installed version on userscript catalog sites" setting does ?

### DISCUSSION THREAD

**Discussion by T1mL3arn** [violentmonkey/violentmonkey#1386] [Q&A] [answered]
The setting "Expose installed version on userscript catalog sites" is in Settings -> General. 

Could someone explain what exactly it does ?

**Comment by tophf** [comment, marked as answer]
The enabled site will be able to ask Violentmonkey which version of the script is installed and show it on the installation page, for example: https://greasyfork.org/en/scripts/369400-local-youtube-downloader

**Reply by T1mL3arn** [reply]
Oh, I figured it out, though screenshot would be more informative.

When the script above is installed and this setting is set:

![image](https://user-images.githubusercontent.com/9349164/135786335-18d6f9f9-a748-4621-a314-c10d9f175519.png)

the same, but the setting is not set:

![image](https://user-images.githubusercontent.com/9349164/135786405-23e22e13-d0e1-4d7b-93f8-dfd7073e80b6.png)

I was confused with english and russian description of the setting. I believe russian translation is completely wrong, I will make a pr to fix it.
